### PR TITLE
Release 2.0.2

### PR DESCRIPTION
**Pull request by birchmd**
- Address logging fix https://github.com/aurora-is-near/aurora-engine/pull/321
- Storage write logging https://github.com/aurora-is-near/aurora-engine/pull/322

### CODE REVIEW

**Review by joshuajbouw — APPROVED** [review]

**Review by mrLSD — APPROVED** [review]

**Review by joshuajbouw — CHANGES_REQUESTED** [review]
The README needs to be updated with deployed versions to each network as 2.0.2

**Review by joshuajbouw — APPROVED** [review]

File: engine/src/lib.rs [hunk 1]
```diff
@@ -225,7 +225,7 @@ mod contract {
             .sdk_expect("ERR_INVALID_ECDSA_SIGNATURE");
 
         #[cfg(feature = "log")]
-        sdk::log(crate::prelude::format!("signer_public_key 0x{}", sender).as_str());
+        sdk::log(crate::prelude::format!("signer_address {:?}", sender).as_str());
```
Comment (artob) [inline engine/src/lib.rs:228]: @birchmd @joshuajbouw This is now not in a consistent format with the new entries, which include a colon. Please harmonize.

Comment (joshuajbouw) [reply engine/src/lib.rs:228]: Display, as it turned out, didn’t show the whole address. Needs debug when we looked into why the address was being concatenated. Includes the 0x already too.

File: CHANGES.md [hunk 2]
```diff
@@ -123,6 +133,8 @@ struct SubmitResult {
 ## [1.0.0] - 2021-05-12
 
 [Unreleased]: https://github.com/aurora-is-near/aurora-engine/compare/2.0.0...master
+[2.0.2]: https://github.com/aurora-is-near/aurora-engine/compare/2.0.0...2.0.2
```
Comment (artob) [inline CHANGES.md:136]: Incorrect

File: engine/src/lib.rs (hunk 1 above)
Comment (joshuajbouw) [reply engine/src/lib.rs:228]: Removed colons from the other messages.

**Review by artob — APPROVED** [review]

**Pull request merged**